proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -74,4 +75,20 @@ message DescribeResponse {
 message DeploymentWorkflowMemo {
     google.protobuf.Timestamp create_time = 1;
     bool is_current_deployment = 2;
```

Comment:
isn't this `effective` now? I might confuse them though.

Reply:
no this is about deployment themselves, not the effective deployment of a particular execution.